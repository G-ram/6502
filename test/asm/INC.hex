a9 07 a2 08 8d 00 01 8e 01 01 ee 00 01 ce 01 01 
