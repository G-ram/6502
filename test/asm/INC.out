A: 07
X: 08
Y: 00
P: 00010000
S: 20
PC: 610
100 : 08
101 : 07
